strongbox-data-service/src/main/java/org/carlspring/strongbox/gremlin/server/GremlinServerConfig.java

```diff
@@ -0,0 +1,43 @@
+package org.carlspring.strongbox.gremlin.server;
+
+import java.io.InputStream;
+
+import org.apache.tinkerpop.gremlin.server.GremlinServer;
+import org.apache.tinkerpop.gremlin.server.Settings;
+import org.janusgraph.core.JanusGraph;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.boot.autoconfigure.condition.ConditionalOnProperty;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.core.io.Resource;
+
+/**
+ * @author Przemyslaw Fusik
+ * @author sbespalov
+ */
+@Configuration
+@ConditionalOnProperty(prefix = "strongbox.graph.gremlin.server", name = "enabled", havingValue = "true")
+public class GremlinServerConfig
+{
+
+    @Bean(destroyMethod = "stop")
+    GremlinServer gremlinServer(JanusGraph janusGraph,
+                                @Value("classpath:/gremlin/gremlin-server.yaml")
+                                Resource gremlinServerConf)
+            throws Exception
```

Comment:
@sbespalov : Is this config file Strongbox-specific, or is it Gremlin-specific? If it's something specifically for our project, we should add a task to document it.

Reply:
this is  Gremlin-specific. I don't think that users need to change it.